src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,32 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int enginePower;
+    private int weight;
+
     public Bulldozer() {
     }
 
     @Override
     public void doWork() {
         System.out.println("Bulldozer started to work");
     }
+
+    public int getEnginePower() {
+        return enginePower;
+    }
+
+    public Bulldozer setEnginePower(int enginePower) {
+        this.enginePower = enginePower;
+        return this;
+    }
+
+    public int getWeight() {
+        return weight;
+    }
+
+    public Bulldozer setWeight(int weight) {
+        this.weight = weight;
+        return this;
+    }
```

Comment:
Do the same for Excavator and Truck classes
1. use constructor-based fields initialization
2. setters should have `void` return type
```suggestion
    public Bulldozer(int enginePower, int weight) {
    this.enginePower = enginePower;
    this.weight = weight;
    }

    @Override
    public void doWork() {
        System.out.println("Bulldozer started to work");
    }

    public int getEnginePower() {
        return enginePower;
    }

    public int getWeight() {
        return weight;
    }
```